packages/compass-indexes/src/components/indexes/indexes.tsx

```diff
@@ -30,29 +31,29 @@ type IndexesProps = {
   isWritable: boolean;
   isReadonly: boolean;
   isReadonlyView: boolean;
-  preferencesReadOnly: boolean;
   description?: string;
   error: string | null;
   localAppRegistry: AppRegistry;
   isRefreshing: boolean;
   sortIndexes: (name: SortColumn, direction: SortDirection) => void;
   refreshIndexes: () => void;
   dropFailedIndex: (id: string) => void;
+  readOnly?: boolean;
 };
 
 export const Indexes: React.FunctionComponent<IndexesProps> = ({
   indexes,
   isWritable,
   isReadonly,
   isReadonlyView,
-  preferencesReadOnly,
   description,
   error,
   localAppRegistry,
   isRefreshing,
   sortIndexes,
   refreshIndexes,
   dropFailedIndex,
+  readOnly, // preferences readOnly.
```

Comment:
Similarly here, it’s totally optional but for function components `usePreference()` might be a bit easier to read than `withPreferences()`, although in the end they pretty much boil down to the same thing anyway